src_types/pages/$w.d.ts

```diff
@@ -1,2 +1,35 @@
-// This is a generated file
-// Run `npm run generate-dts` to generate it
\ No newline at end of file
+declare type IntersectionArrayAndBase<T, P> = {
+    [K in keyof T]: K extends P ? T[K] : T[K] & T[K][];
+};
+
+declare type IsWixElementSelector<S> = S extends keyof WixElementSelector ? WixElementSelector[S] : never;

```

Comment:
Shouldn't the fallback for this be a string? (instead of never)
Cause then in line 8 $w excepts either `T | never` (when s doesn't extend keyof WixElementSelector)

Reply:
no. if it's will be a string, no validation on any string will work. I am returning never and that's triggers the error when a string doesn't match.